tests/macros/utils.js

```diff
@@ -191,12 +385,54 @@ function lintHTML(html, fragment = true) {
     }
 }
 
+/**
+ * Reads a generic fixture file from the `fixtures` directory.
+ *
+ * @param {string|string[]} filePath
+ *        A path to a file relative to the `fixtures` directory.
+ * @param {string|null} [encoding]
+ *        The file encoding. Set to `null` to load the file as a binary `Buffer`.
+ *
+ * @returns {string}
```

Comment:
Unfortunately, JSDoc doesn’t provide any way to define function overloads, so&nbsp;the&nbsp;return type will always show up&nbsp;as&nbsp;a&nbsp;`string`, unless we create `utils.d.ts` or&nbsp;migrate this file to&nbsp;TypeScript.